pkg/agent/apiserver/handlers/ovstracing/handler.go

```diff
@@ -380,7 +380,9 @@ func HandleFunc(aq querier.AgentQuerier) http.HandlerFunc {
 				// ovs-appctl has been executed but returned an error (e.g. the provided
 				// "flow" expression is incorrect). Return the error output to the client in
 				// this case.
-				out = execErr.GetErrorOutput()
+				klog.Errorf("Tracing command failed: %s", execErr.GetErrorOutput())
```

Comment:
we use structured logging (`klog.ErrorS`). You can update other logs in this function as well.

Reply:
Updated.